string muscle # 0 to 3
string action # `move to`, `move with` or `keep`
float32 setpoint # radians, radians/sec or newtons
---
bool is_success
